jujugui/static/gui/src/app/utils/bakery-utils.js

```diff
@@ -64,12 +63,15 @@ YUI.add('bakery-utils', function(Y) {
     @param {Function} charmstoreCookieSetter A function that can be used to
       send macaroons to the charm store so that they are stored as cookies.
     @param {Object} webHandler The HTTP client that will be used by the bakery.
+    @param {Object} BakeryStorage A reference to the BakeryStorage constructor.
```

Comment:
+1 I don't think this is a place where we need that flexability, esp as the create method is still basically bound to creating the bakery we have, rather than a generic 'bakery'.